src/chapters/strings/encoding-characters.rst

```diff
@@ -0,0 +1,119 @@
+
+In this way, the bits in a computer can be viewed as integers. If you want to represent values greater than 255, just use more bits! 
+
+Representing Strings
+--------------------
+
+Strings are collections of characters, so if we can represent each character as a number, then we'll have a way to go from a string to a collection of bits, and back again.
+
+Character Encodings
+^^^^^^^^^^^^^^^^^^^
+
+.. index::
+   pair: character; encoding
+
+Unlike the natrual translation between binary and decimal numbers, there is no natural translation between integers and characters. For example, you might create a pairing of 0 to ``a``, 1 to ``b``, and so on. But what integer should be paired with ``$`` or a tab? Since there is no natural way to translate between characters and integers, computer scientists have had to make such translations up. Such translations are called **character encodings**.
+
+.. index:: Unicode
+
+There are many different encodings, some of which continue to evolve as the way in which we use data evolves. For instance, the most recent versions of the Unicode standard include emoji characters, such as 🌮.
+
+The ASCII Encoding
+^^^^^^^^^^^^^^^^^^
+
+.. index:: ASCII
+
+Most of the characters that you are used to using---including letters, numbers, whitespace, punctuation, and symbols---are part of the **ASCII** (pronounced *ASS-kee*) character encoding. This standard has changed very little since the 1960s, and is a subset of all other commonly-used encodings. In other words, other encodings start with ASCII and built up to include other characters.
+
+.. note:: ASCII stands for American Standard Code for Information Interchange, but most programmers never remember that, so you shouldn't try to either.
+
+ASCII provides a standard translation of the most commonly-used characters to one of the integers 0...127, which means each character can be stored in a computer using a single byte. 
+
+.. index::
+   single: ASCII; table
+
+ASCII maps ``a`` to 97, ``b`` to 98, and so on for lowercase letters, with ``z`` mapping to 122. Uppercase letters map to the values 65 through 90. The other integers between 0 and 127 represent symbols, punctuation, and other assorted odd characters. This scheme is called the **ASCII table**, and rather than replicate it here, we refer you to an `excellent one online <https://www.ascii-code.com/>`_.
+
+In summary, strings are stored in a computer using the following process:
+
+#. Break a string into its individual characters.
+#. Use a character encoding, such as ASCII, to convert each of the characters to an integer.
+#. Convert each integer to a series of bits using decimal-to-binary integer conversion.
+
+.. note:: While JavaScript uses the UTF-16 encoding, we will rarely need anything outside of its ASCII subset. Thus, we'll usually talk about "ASCII codes" in JavaScript, even though it technicall supports a broader ranges of character codes.
```

Comment:
I understand the desire for technical accuracy, but I think this note is extraneous.  This is especially true given the "rarely need" qualifier.  After hitting the students with the encoding information, I think the note is one detail too many.

If we keep the note, fix the "technicall" typo.

Reply:
I'm going to make this a "Fun Fact," which should more clearly indicate the optional nature. I'll also improve the wording to make this more clear. I like including this because informs students that there is a wide variety of character sets.

Reply:
UTF-16 = fun?  I'm not sure I see the connection, but I'll defer to your judgement.